docs/debugging.md

```diff
@@ -16,18 +55,44 @@ This enables you to then execute the compiled binary via Delve, pass any argumen
 dlv exec $GOBIN/terraform --headless --listen :2345 --log -- apply
```

Comment:
Nit: This PR did not make the change, but perhaps we could use this opportunity to correct it. Some configuration do not have $GOBIN set, thus this would not be a valid path. Could we instead just put a placeholder like "PATH_TO_TERRAFORM_BINARY"

Reply:
I've updated the line to use the default value `$HOME/go/bin`, which should be copy-pastable by most users and doesn't require prior setup like $GOBIN does. Hopefully the comment is sufficient for any users who _have_ configured a non-default path using those ENVs.